ui/config/deprecation-workflow.js

```diff
@@ -3,20 +3,17 @@ self.deprecationWorkflow = self.deprecationWorkflow || {};
 
 self.deprecationWorkflow.config = {
   workflow: [
-    { handler: 'silence', matchId: 'ember-global' },
-    { handler: 'silence', matchId: 'ember.component.reopen' },
     { handler: 'silence', matchId: 'ember-keyboard.first-responder-inputs' },
-    { handler: 'silence', matchId: 'implicit-injections' },
     { handler: 'silence', matchId: 'ember-test-waiters-legacy-module-name' },
     { handler: 'silence', matchId: 'ember-keyboard.old-propagation-model' },
     {
       handler: 'silence',
-      matchId: 'deprecated-run-loop-and-computed-dot-access',
+      matchId: 'ember.built-in-components.legacy-arguments',
```

Comment:
should this still be silenced?

Reply:
I meant, should we throw here instead of silence?

Reply:
I wouldnt recommend setting these to throw unless you are working locally and trying to see the errors halt your test suite which is what I did. If you miss a spot with a deprecation and set it to throw, it can blow the app on code that works perfectly fine (the case here since I am not upgrading to ember 4 in this PR)

Deprecations will still be logged from ember even with these entries set to silence or not even here, they will just be spewed and not grouped.